be/src/column/json_column.h

```diff
@@ -68,6 +68,8 @@ class JsonColumn final : public ColumnFactory<ObjectColumn<JsonValue>, JsonColum
 
     void resize(size_t n) override;
 
+    void reserve(size_t n) override{};
```

Comment:
why override and set it empty ?

Reply:
flat json don't need use ObjectPool, so we can't reserve immediately after create a column, and reserve object doesn't have a significant performance improve on json type